src/rcc.rs

```diff
@@ -28,7 +30,31 @@ impl RccExt for RCC {
             crrcr: CRRCR { _0: () },
             cfgr: CFGR {
                 hclk: None,
-                hsi48: false,
```

Comment:
HSI48, should stay as it is a valid clock for other L4 chips, and calling it usb48 doesnt quite fit in my mind as it can be used for rng, sdio etc.

Reply:
What about naming it clk48 ? The idea here is that  some MCU does not have the HSI48 clock. A more generic naming would allow use to have the same user code and be able to run it on different L4 MCU.

Reply:
Also not that HSI48 is used only for USB, RNG SDMMC clock or MCO output. It look to me that it has been design for USB,RNG SDMMC purose in mind.

Reply:
I don't really like the idea of treating hsi48 and msi as one 48mhz field, I think the `clocks` returned from the `rcc` should be as close to what the hardware is as possible, and then it should be left up to the peripherals to check it either has hsi48 or msi or pll etc when initializing.

Reply:
Ok, so to me there is input clock and output clock. Maybe the idea is to  configure a pair of clock:

(in, out, [scale]) ?

may be for now I will just feature gate the HSI48  and add the MSI clock.

Reply:
> may be for now I will just feature gate the HSI48 and add the MSI clock.

That sounds good to me :).